### DISCUSSION TITLE
Daily Scrum Meeting #3

### DISCUSSION THREAD

**Discussion by IrfanZnA** [GIA-003/Kado#23] [General]
meeting 3

**Comment by Rizkiblack** [comment]
kami sudah meminta pendapat dari beberapa teman mengenai kado apakah sudah bagus diberikan kepada client yaitu adiknya yang seorang perempuan, dan mendapatkan respon positif

**Comment by HudaGates** [comment]
Saya sudah mengirimkan rincian untuk estimasi biaya barang yang dibuat untuk membuat kado

**Comment by IrfanZnA** [comment]
baiklah teman" karna sudah bekerja keras dan kooperatif, sisanya kita tinggal memberikan kado kepada product owner dan menunggu responya
